girder/models/upload.py

```diff
@@ -355,7 +355,7 @@ def createUpload(self, user, name, parentType, parent, size, mimeType=None,
             upload['reference'] = reference
 
         if parentType and parent:
-            upload['parentType'] = parentType.lower()
```

Comment:
In that case, we should update the docstring of this function and mention that it accepts a list/tuple with model and plugin.